config/config.php

```diff
@@ -21,6 +21,14 @@
         'version' => '1.0.0',
 
         /*
+         * Set this to "true" if the app is in production.
+         *
+         * If set to true, the 'build' & 'rename' commands won't fire
```

Comment:
Replace with: If true, development commands won't be available as the app will be in the production environment.